### DISCUSSION TITLE
Can you update GBB flags post Linux install with a SuzyQable?

### DISCUSSION THREAD

**Discussion by ITJoeSchmo** [velvet-os/imagebuilder#278] [Q&A] [answered]
I didn't realize the guide suggested doing this first, and have my Lenovo Duet 5 like 99% setup now before realizing :(.

I noticed in pmOS wiki they mention how to update them from Linux:
https://wiki.postmarketos.org/wiki/Category:ChromeOS

Then doing some additional digging found this post, which confirms you can use the SuzyQable from Linux, but they were missing gsctools to enable CCD to begin with:    https://forum.chrultrabook.com/t/disable-firmware-write-protect-from-linux-using-suzyq-cable/2127 

According to the last post you can grab gsctools binary for gnu here:  https://files.tree123.org/utils/arm64/gnu/

(there are some other ChromeOS bins in there also)

I will try it and update how it goes. 

Update: got ahead of myself, was able to unlock CCD but I didn't realize that also disables developer mode, so I got in a boot loop. Used a recovery image, and confirmed it was able to open CCD via Linux -- maybe there is a way to do it if we can find a way around the disabling dev mode. 

Going to have to start from scratch on my install lol. No worries though.

**Comment by ITJoeSchmo** [comment]
Fuck me, be careful I wasn't thinking about how gsctools -a -o disables developer mode, so while it seems to have enables CCD I think, I am now stuck because developer mode was disabled during the reboot. I can get to recovery, but I don't want to enable it again as I think it'll wipe my Linux install on the drive (or maybe it's already wiped?) hoping I haven't completely bricked this :( will post an update

**Comment by thenameisluk** [comment]
@ITJoeSchmo damn chill, enabling developer mode again won't wipe your device since there is no chrome os to do that

and bios doesn't wipe your device, so you can safely enable the developer mode again.

but not sure if it will boot unsigned kernel (your linux), instead of rushing everything you could actually wait for sb to reply?

**Reply by ITJoeSchmo** [reply]
Yeah you're not wrong. I used an exploit called BadRecovery to get a shell in recovery mode but I couldn't seem to run the needed commands to re-enable booting unsigned kernel/etc. Ended up just actually restoring Chrome OS.

**Reply by thenameisluk** [reply, marked as answer]
@ITJoeSchmo if yo could just wait for my reply, i could let you know there is that thing called [sh1mmer](https://sh1mmer.me/) you could have used to restore usb boot without having to restore chromeos
